NewForce-Cohort-9/nutshell-react-jurassic-attack
# Task Card and Checkbox

# Added

-Added TaskCreateForm.jsx to preform the create new function
- Added the usable checkbox that will remove task from list when completed


# Testing Instructions

1. After logging in, go to Task and Events tab on NavBar
2. Click Create New Task, enter a task and submit
3. You should see you new task
4. Check the box marked Done
5. See your task disappear from the page
6. Check database that your task is there and the competed object says true >> ["completed": true]
